async.js

```diff
@@ -13,11 +13,49 @@ const isStar = true;
  * @returns {Promise<Array>}
  */
 function runParallel(jobs, parallelNum, timeout = 1000) {
-    // асинхронная магия
+    let results = [];
+
+    return new Promise(
+        resolve => {
+            if (!jobs.length) {
+                resolve(results);
+
+                return;
```

Comment:
А чем это будет отличаться от
```suggestion
                return resolve(results);
```
?